ntbs-service/wwwroot/css/home.scss

```diff
@@ -12,3 +12,33 @@
     max-width: 200px;
     margin: 5px 0 auto 15px
 }
+
+.homepage-kpi-card {
+    background-color: $phe-red;

```

Comment:
Have the visuals for this been run past nancy? Just a quick screenshot would probably do.

Reply:
Yes, double checked with Tehreem and Nancy